frontend/src/assets/css/scroll-bar.scss

```diff
@@ -0,0 +1,18 @@
+/* Works on Firefox */
+html {
+  scrollbar-color: #3563ad rgba(238 239 242 / 50%);
```

Comment:
Maybe we should put these colors in `vars.scss`? Especially `#3563ad` is repeated twice here.

Reply:
I thought about it, but we already have 6 types of blue, and the same with gray. In our layout, we use these colors just once. I think we should not clutter vars with non-reusable variables